src-tauri/src/lib.rs

```diff
@@ -28,15 +38,21 @@ pub fn run() {
         .plugin(tauri_plugin_store::Builder::default().build())
         .setup(|app| {
             let notes_store = app.handle().store_builder("notes.bin").build();
-
             let local_note_service = LocalNoteService::new(notes_store);
             let _remote_note_service =
                 RemoteNoteService::new("/notes".to_string(), HttpClientImpl::default());
 
+            let user_store = app.handle().store_builder("users.bin").build();
+            let local_user_service = LocalUserService::new(user_store);
+            let _remote_user_service =
+                RemoteUserService::new("/users".to_string(), HttpClientImpl::default());
+
             let note_manager = NoteManager::new(Box::new(local_note_service));
+            let user_manager = UserManager::new(Box::new(local_user_service));
 
             let state = AppState {
                 notes_manager: note_manager,
+                user_manager,
```

Comment:
We should probably have a LocalAppState and RemoteAppState that we can directly inject, instead of defining it here

This would also remove half the code of this function, defining only LocalAppState and RemoteAppState.

Reply:
True, created #45 in that sense.